### ISSUE TITLE
[0.9.6] influxdb randomly crashing with panic: runtime error: index out of range

### ISSUE THREAD

**Issue by mlosev** [influxdb/influxdb#5075]
```
2015/12/10 00:44:42 InfluxDB starting, version 0.9.6, branch 0.9.6, commit ea8bbabfb68394aab5d9f23f08e29986bc137955, built 2015-12-09T16:10:21+0000
2015/12/10 00:44:42 Go version go1.4.2, GOMAXPROCS set to 12
2015/12/10 00:44:43 Using configuration at: /config/config.toml
[metastore] 2015/12/10 00:44:43 Using data dir: /data/meta
[metastore] 2015/12/10 00:44:43 Skipping cluster join: already member of cluster: nodeId=1 raftEnabled=true peers=[influx-mysql-cp.beget:8088]
[metastore] 2015/12/10 00:44:43 Node at influx-mysql-cp.beget:8088 [Follower]
[metastore] 2015/12/10 00:44:48 Node at influx-mysql-cp.beget:8088 [Leader]. peers=[influx-mysql-cp.beget:8088]
[metastore] 2015/12/10 00:44:48 Updated node id=1 hostname=influx-mysql-cp.beget:8088
[metastore] 2015/12/10 00:44:48 spun up monitoring for 1
[store] 2015/12/10 00:44:48 Using data dir: /data/db
[tsm1wal] 2015/12/10 00:44:48 tsm1 WAL starting with 10485760 segment size
[tsm1wal] 2015/12/10 00:44:48 tsm1 WAL writing to /data/wal/_internal/monitor/2
[cacheloader] 2015/12/10 00:44:48 reading file /data/wal/_internal/monitor/2/_00001.wal, size 859
[cacheloader] 2015/12/10 00:44:48 reading file /data/wal/_internal/monitor/2/_00004.wal, size 2555
[cacheloader] 2015/12/10 00:44:48 reading file /data/wal/_internal/monitor/2/_00005.wal, size 7824
[cacheloader] 2015/12/10 00:44:48 reading file /data/wal/_internal/monitor/2/_00007.wal, size 6363
[cacheloader] 2015/12/10 00:44:48 reading file /data/wal/_internal/monitor/2/_00009.wal, size 830
[cacheloader] 2015/12/10 00:44:48 reading file /data/wal/_internal/monitor/2/_00010.wal, size 831
[cacheloader] 2015/12/10 00:44:48 reading file /data/wal/_internal/monitor/2/_00011.wal, size 827
[cacheloader] 2015/12/10 00:44:48 reading file /data/wal/_internal/monitor/2/_00012.wal, size 3450
[cacheloader] 2015/12/10 00:44:48 reading file /data/wal/_internal/monitor/2/_00013.wal, size 3591
[cacheloader] 2015/12/10 00:44:48 reading file /data/wal/_internal/monitor/2/_00015.wal, size 4508
[cacheloader] 2015/12/10 00:44:48 reading file /data/wal/_internal/monitor/2/_00016.wal, size 773
[cacheloader] 2015/12/10 00:44:48 reading file /data/wal/_internal/monitor/2/_00017.wal, size 6379
[cacheloader] 2015/12/10 00:44:48 reading file /data/wal/_internal/monitor/2/_00023.wal, size 4488
[cacheloader] 2015/12/10 00:44:48 reading file /data/wal/_internal/monitor/2/_00025.wal, size 8472
[cacheloader] 2015/12/10 00:44:48 reading file /data/wal/_internal/monitor/2/_00040.wal, size 0
[tsm1wal] 2015/12/10 00:44:48 tsm1 WAL starting with 10485760 segment size
[tsm1wal] 2015/12/10 00:44:48 tsm1 WAL writing to /data/wal/_internal/monitor/3
[cacheloader] 2015/12/10 00:44:48 reading file /data/wal/_internal/monitor/3/_00001.wal, size 957
[cacheloader] 2015/12/10 00:44:48 reading file /data/wal/_internal/monitor/3/_00002.wal, size 3548
[cacheloader] 2015/12/10 00:44:48 reading file /data/wal/_internal/monitor/3/_00004.wal, size 5442
[cacheloader] 2015/12/10 00:44:48 reading file /data/wal/_internal/monitor/3/_00005.wal, size 781
[cacheloader] 2015/12/10 00:44:48 reading file /data/wal/_internal/monitor/3/_00006.wal, size 7095
[cacheloader] 2015/12/10 00:44:48 reading file /data/wal/_internal/monitor/3/_00007.wal, size 838
[cacheloader] 2015/12/10 00:44:48 reading file /data/wal/_internal/monitor/3/_00009.wal, size 3609
[cacheloader] 2015/12/10 00:44:48 reading file /data/wal/_internal/monitor/3/_00014.wal, size 7284
[cacheloader] 2015/12/10 00:44:48 reading file /data/wal/_internal/monitor/3/_00015.wal, size 1760
[cacheloader] 2015/12/10 00:44:48 reading file /data/wal/_internal/monitor/3/_00016.wal, size 0
[tsm1wal] 2015/12/10 00:44:48 tsm1 WAL starting with 10485760 segment size
[tsm1wal] 2015/12/10 00:44:48 tsm1 WAL writing to /data/wal/mysql-cp/default/1
[cacheloader] 2015/12/10 00:44:49 reading file /data/wal/mysql-cp/default/1/_00052.wal, size 2381471
[cacheloader] 2015/12/10 00:44:50 reading file /data/wal/mysql-cp/default/1/_00053.wal, size 8730030
[cacheloader] 2015/12/10 00:44:51 reading file /data/wal/mysql-cp/default/1/_00054.wal, size 0
[handoff] 2015/12/10 00:45:24 Starting hinted handoff service
[monitor] 2015/12/10 00:45:24 'hh' registered for diagnostics monitoring
[handoff] 2015/12/10 00:45:24 Using data dir: /data/hh
[subscriber] 2015/12/10 00:45:24 opened service
[monitor] 2015/12/10 00:45:24 Starting monitor system
[monitor] 2015/12/10 00:45:24 'build' registered for diagnostics monitoring
[monitor] 2015/12/10 00:45:24 'runtime' registered for diagnostics monitoring
[monitor] 2015/12/10 00:45:24 'network' registered for diagnostics monitoring
[monitor] 2015/12/10 00:45:24 'system' registered for diagnostics monitoring
[cluster] 2015/12/10 00:45:24 Starting cluster service
[shard-precreation] 2015/12/10 00:45:24 Starting precreation service with check interval of 1m0s, advance period of 1h0m0s
[snapshot] 2015/12/10 00:45:24 Starting snapshot service
[copier] 2015/12/10 00:45:24 Starting copier service
[admin] 2015/12/10 00:45:24 Starting admin service
[monitor] 2015/12/10 00:45:24 Storing statistics in database '_internal' retention policy '', at interval 50s
[admin] 2015/12/10 00:45:24 Listening on HTTP: 192.168.0.36:8083
[continuous_querier] 2015/12/10 00:45:24 Starting continuous query service
[httpd] 2015/12/10 00:45:24 Starting HTTP service
[httpd] 2015/12/10 00:45:24 Authentication enabled: false
[httpd] 2015/12/10 00:45:24 Listening on HTTP: 192.168.0.36:8086
[udp] 2015/12/10 00:45:24 Started listening on UDP: 192.168.0.36:4444
[retention] 2015/12/10 00:45:24 Starting retention policy enforcement service with check interval of 30m0s
[run] 2015/12/10 00:45:24 Listening for signals
2015/12/10 00:45:24 Sending anonymous usage statistics to m.influxdb.com
panic: runtime error: index out of range

goroutine 314 [running]:
github.com/influxdb/influxdb/tsdb/engine/tsm1.(*WriteWALEntry).Encode(0xccd5dda0f8, 0xc9ffe28000, 0x100000, 0x100000, 0x0, 0x0, 0x0, 0x0, 0x0)
        /tmp/tmp.QrWAFdTR6B/src/github.com/influxdb/influxdb/tsdb/engine/tsm1/wal.go:373 +0x1234
github.com/influxdb/influxdb/tsdb/engine/tsm1.(*WAL).writeToLog(0xc208c49100, 0x7f0799e68b40, 0xccd5dda0f8, 0x0, 0x0, 0x0)
        /tmp/tmp.QrWAFdTR6B/src/github.com/influxdb/influxdb/tsdb/engine/tsm1/wal.go:208 +0xbb
github.com/influxdb/influxdb/tsdb/engine/tsm1.(*WAL).WritePoints(0xc208c49100, 0xc233613080, 0x0, 0x0, 0x0)
        /tmp/tmp.QrWAFdTR6B/src/github.com/influxdb/influxdb/tsdb/engine/tsm1/wal.go:149 +0xa3
github.com/influxdb/influxdb/tsdb/engine/tsm1.(*DevEngine).WritePoints(0xc208058960, 0xc31fdec000, 0x3d4, 0x400, 0x0, 0xcba5e16000, 0x3d4, 0x400, 0x0, 0x0)
        /tmp/tmp.QrWAFdTR6B/src/github.com/influxdb/influxdb/tsdb/engine/tsm1/engine.go:255 +0x55f
github.com/influxdb/influxdb/tsdb.(*Shard).WritePoints(0xc208c1f340, 0xc31fdec000, 0x3d4, 0x400, 0x0, 0x0)
        /tmp/tmp.QrWAFdTR6B/src/github.com/influxdb/influxdb/tsdb/shard.go:263 +0x66d
github.com/influxdb/influxdb/tsdb.(*Store).WriteToShard(0xc2080e6000, 0x1, 0xc31fdec000, 0x3d4, 0x400, 0x0, 0x0)
        /tmp/tmp.QrWAFdTR6B/src/github.com/influxdb/influxdb/tsdb/store.go:398 +0x14e
github.com/influxdb/influxdb/cluster.func·002(0x1, 0x1, 0xc31fdec000, 0x3d4, 0x400)
        /tmp/tmp.QrWAFdTR6B/src/github.com/influxdb/influxdb/cluster/points_writer.go:315 +0x131
created by github.com/influxdb/influxdb/cluster.(*PointsWriter).writeToShard
        /tmp/tmp.QrWAFdTR6B/src/github.com/influxdb/influxdb/cluster/points_writer.go:347 +0x31a

goroutine 1 [chan receive, 3 minutes]:
main.(*Main).Run(0xc208049700, 0xc20800a010, 0x1, 0x1, 0x0, 0x0)
        /tmp/tmp.QrWAFdTR6B/src/github.com/influxdb/influxdb/cmd/influxd/main.go:96 +0x7b3
main.main()
        /tmp/tmp.QrWAFdTR6B/src/github.com/influxdb/influxdb/cmd/influxd/main.go:46 +0xdc

...
```

influxdb was downloaded from https://s3.amazonaws.com/influxdb/influxdb_0.9.6_x86_64.tar.gz

**Comment by mlosev** [comment]
it seems that influxdb is crashing on writes

after enabling write logs I can see data being written

```
mysql_cp,customer=blond,hostname=tsoi,user=blond_drup2 access_denied=0i,busy_time=0i,commit_transactions=0i,concurrent_connections=0i,connected_time=1i,cpu_time=0i,denied_connections=0i,empty_q
ueries=29i,lost_connections=0i,other_commands=4i,rollback_transactions=0i,rows_fetched=612i,rows_updated=62i,select_commands=128i,table_rows_read=1967i,total_connections=2i,update_commands=101i
 1449709200000000000
mysql_cp,customer=cpaset4v,hostname=tsoi,user=cpaset4v_youne11 access_denied=0i,busy_time=0i,commit_transactions=0i,concurrent_connections=0i,connected_time=0i,cpu_time=0i,denied_connections=0i
,empty_queries=159i,lost_connections=0i,other_commands=19i,rollback_transactions=0i,rows_fetched=2269i,rows_updated=10i,select_commands=274i,table_rows_read=6985i,total_connections=4i,update_co
mmands=11i 1449709200000000000
mysql_cp,customer=tkgery98,hostname=tsoi,user=tkgery98_germes access_denied=0i,busy_time=0i,commit_transactions=0i,concurrent_connections=0i,connected_time=0i,cpu_time=0i,denied_connections=0i,
empty_queries=9i,lost_connections=0i,other_commands=1i,rollback_transactions=0i,rows_fetched=277i,rows_updated=0i,select_commands=15i,table_rows_read=350i,total_connections=1i,update_commands=0
i 1449709200000000000
mysql_cp,customer=efremo8j,hostname=tsoi,user=efremo8j_prodl36 access_denied=0i,busy_time=0i,commit_transactions=0i,concurrent_connections=0i,connected_time=0i,cpu_time=0i,denied_connections=0i
,empty_queries=46i,lost_connections=0i,other_commands=1i,rollback_transactions=0i,rows_fetched=247i,rows_updated=0i,select_commands=261i,table_rows_read=1145i,total_connections=1i,update_comman
ds=0i 1449709200000000000
mysql_cp,customer=mydenzuv,hostname=tsoi,user=mydenzuv_nzen access_denied=0i,busy_time=0i,commit_transactions=0i,concurrent_connections=0i,connected_time=8i,cpu_time=0i,denied_connections=0i,em
pty_queries=27i,lost_connections=0i,other_commands=4i,rollback_transactions=0i,rows_fetched=920i,rows_updated=27i,select_commands=90i,table_rows_read=1211i,total_connections=3i,update_commands=
31i 1449709200000000000
panic: runtime error: index out of range

goroutine 155 [running]:
github.com/influxdb/influxdb/tsdb/engine/tsm1.(*WriteWALEntry).Encode(0xc208904250, 0xc2313ea000, 0x100000, 0x100000, 0x0, 0x0, 0x0, 0x0, 0x0)
        /tmp/tmp.QrWAFdTR6B/src/github.com/influxdb/influxdb/tsdb/engine/tsm1/wal.go:373 +0x1234
github.com/influxdb/influxdb/tsdb/engine/tsm1.(*WAL).writeToLog(0xc2088e7e80, 0x7fe1ac08a5b0, 0xc208904250, 0x0, 0x0, 0x0)
        /tmp/tmp.QrWAFdTR6B/src/github.com/influxdb/influxdb/tsdb/engine/tsm1/wal.go:208 +0xbb
github.com/influxdb/influxdb/tsdb/engine/tsm1.(*WAL).WritePoints(0xc2088e7e80, 0xc21a3e2ba0, 0x0, 0x0, 0x0)
        /tmp/tmp.QrWAFdTR6B/src/github.com/influxdb/influxdb/tsdb/engine/tsm1/wal.go:149 +0xa3
github.com/influxdb/influxdb/tsdb/engine/tsm1.(*DevEngine).WritePoints(0xc208968960, 0xc244208000, 0x351, 0x400, 0x0, 0xc22c32a000, 0x351, 0x400, 0x0, 0x0)
        /tmp/tmp.QrWAFdTR6B/src/github.com/influxdb/influxdb/tsdb/engine/tsm1/engine.go:255 +0x55f
github.com/influxdb/influxdb/tsdb.(*Shard).WritePoints(0xc2089386e0, 0xc244208000, 0x351, 0x400, 0x0, 0x0)
        /tmp/tmp.QrWAFdTR6B/src/github.com/influxdb/influxdb/tsdb/shard.go:263 +0x66d
github.com/influxdb/influxdb/tsdb.(*Store).WriteToShard(0xc2080e4000, 0x1, 0xc244208000, 0x351, 0x400, 0x0, 0x0)
        /tmp/tmp.QrWAFdTR6B/src/github.com/influxdb/influxdb/tsdb/store.go:398 +0x14e
github.com/influxdb/influxdb/cluster.func·002(0x1, 0x1, 0xc244208000, 0x351, 0x400)
        /tmp/tmp.QrWAFdTR6B/src/github.com/influxdb/influxdb/cluster/points_writer.go:315 +0x131
created by github.com/influxdb/influxdb/cluster.(*PointsWriter).writeToShard
        /tmp/tmp.QrWAFdTR6B/src/github.com/influxdb/influxdb/cluster/points_writer.go:347 +0x31a

```

how to debug this situation?

**Comment by mlosev** [comment]
0.9.5.1 runs stable on the same kind of data

**Comment by mlosev** [comment]
same crashes began on another 0.9.6 instance (clean install without previous data)

writing dataset is different

```
mysql_quota,customer=isever8k,hostname=rembo,user=isever8k_ivan-mysql blocks=11145216i,files=140i 1449715200000000000
mysql_quota,customer=zin13689,hostname=rembo,user=zin13689_p-mysql blocks=10215424i,files=166i 1449715200000000000
mysql_quota,customer=dimat8,hostname=rembo,user=dimat8_jm2-mysql blocks=23437312i,files=217i 1449715200000000000
mysql_quota,customer=amakse,hostname=rembo,user=amakse_ocart-mysql blocks=13348864i,files=234i 1449715200000000000
mysql_quota,customer=rofanddn,hostname=rembo,user=rofanddn_b-mysql blocks=785924096i,files=1180i 1449715200000000000
mysql_quota,customer=askaro,hostname=rembo,user=askaro_opca-mysql blocks=34291712i,files=410i 1449715200000000000
mysql_quota,customer=grigor0u,hostname=rembo,user=grigor0u_jm1-mysql blocks=3842048i,files=235i 1449715200000000000
mysql_quota,customer=gretany5,hostname=rembo,user=gretany5_prigoto-mysql blocks=2568192i,files=24i 1449715200000000000
panic: runtime error: index out of range

goroutine 382 [running]:
github.com/influxdb/influxdb/tsdb/engine/tsm1.(*WriteWALEntry).Encode(0xc2088ca030, 0xc226b84000, 0x100000, 0x100000, 0x0, 0x0, 0x0, 0x0, 0x0)
        /go/src/github.com/influxdb/influxdb/tsdb/engine/tsm1/wal.go:373 +0x1234
github.com/influxdb/influxdb/tsdb/engine/tsm1.(*WAL).writeToLog(0xc208976000, 0x7f3dc019d280, 0xc2088ca030, 0x0, 0x0, 0x0)
        /go/src/github.com/influxdb/influxdb/tsdb/engine/tsm1/wal.go:208 +0xbb
github.com/influxdb/influxdb/tsdb/engine/tsm1.(*WAL).WritePoints(0xc208976000, 0xc2219f8300, 0x0, 0x0, 0x0)
        /go/src/github.com/influxdb/influxdb/tsdb/engine/tsm1/wal.go:149 +0xa3
github.com/influxdb/influxdb/tsdb/engine/tsm1.(*DevEngine).WritePoints(0xc208978000, 0xc22d202000, 0x1b87, 0x1e00, 0x0, 0xc22c616000, 0x1b87, 0x1c00, 0x0, 0x0)
        /go/src/github.com/influxdb/influxdb/tsdb/engine/tsm1/engine.go:255 +0x55f
github.com/influxdb/influxdb/tsdb.(*Shard).WritePoints(0xc208974000, 0xc22d202000, 0x1b87, 0x1e00, 0x0, 0x0)
        /go/src/github.com/influxdb/influxdb/tsdb/shard.go:263 +0x66d
github.com/influxdb/influxdb/tsdb.(*Store).WriteToShard(0xc2080fe000, 0x1, 0xc22d202000, 0x1b87, 0x1e00, 0x0, 0x0)
        /go/src/github.com/influxdb/influxdb/tsdb/store.go:398 +0x14e
github.com/influxdb/influxdb/cluster.func·002(0x1, 0x1, 0xc22d202000, 0x1b87, 0x1e00)
        /go/src/github.com/influxdb/influxdb/cluster/points_writer.go:315 +0x131
created by github.com/influxdb/influxdb/cluster.(*PointsWriter).writeToShard
        /go/src/github.com/influxdb/influxdb/cluster/points_writer.go:347 +0x31a

goroutine 1 [chan receive, 1 minutes]:
main.(*Main).Run(0xc208049700, 0xc20800a010, 0x1, 0x1, 0x0, 0x0)
        /go/src/github.com/influxdb/influxdb/cmd/influxd/main.go:96 +0x7b3
main.main()
        /go/src/github.com/influxdb/influxdb/cmd/influxd/main.go:46 +0xdc

```

**Comment by otoolep** [comment]
Thanks for the report @mlosev -- we'll take a look.

**Comment by mlosev** [comment]
@otoolep @jwilder thank you for fast reaction

I have just built binary from commit 639f7f4bec98e39f47e2bac60f577b23b42910f8
to see if this fixing the issue.

couple of hours ago I was using a binary from master branch but it was crashing almost every write (quick glance through logs)

btw, for the 1st instance I rewrote collector to make dataset with single-field point in seria
```
mysql_cp_update_commands,customer=erofeyiv,hostname=yoda,user=erofeyiv_27 value=43i 1449730200000000000
mysql_cp_update_commands,customer=sasav7,hostname=yoda,user=sasav7_pristan value=1i 1449730200000000000
mysql_cp_update_commands,customer=ccecana,hostname=yoda,user=ccecana_wp2 value=8i 1449730200000000000
mysql_cp_update_commands,customer=oninalwp,hostname=yoda,user=oninalwp_sl value=11i 1449730200000000000
mysql_cp_update_commands,customer=gpskama,hostname=yoda,user=gpskama_1 value=2i 1449730200000000000
mysql_cp_update_commands,customer=masterqw,hostname=yoda,user=masterqw_0_dmkr value=2i 1449730200000000000
mysql_cp_update_commands,customer=skoroxjn,hostname=yoda,user=skoroxjn_dbs value=10i 1449730200000000000
mysql_cp_update_commands,customer=olegk,hostname=yoda,user=olegk_lroldpr value=1i 1449730200000000000
mysql_cp_update_commands,customer=marselniz,hostname=yoda,user=marselniz_autoot value=35i 1449730200000000000
[http] 2015/12/10 06:55:03 192.168.2.78 - root [10/Dec/2015:06:55:03 +0300] POST /write?db=mysql-cp HTTP/1.1 204 0 - python-requests/2.7.0 CPython/2.7.9 Linux/4.3.0-3-beget-acl cb2a6eac-9ef1-11e5-a99e-000000000000 11.625737ms
```
and it still working (4 hours) =)
the problem seems to appear for multi-field points only

**Comment by jwilder** [comment]
@mlosev Does the fix work with your original input or only the modified one?

**Comment by mlosev** [comment]
@jwilder  the situation is as follows:

##### instance 1:
influxdb 0.9.6 (as downloaded from official site)
input data modified to have 1-field points in seria, eg. 
```
mysql_cp_update_commands,customer=erofeyiv,hostname=yoda,user=erofeyiv_27 value=43i 1449730200000000000
```
uptime is 6 hours after changing input data layout
```
root@0edfceb9cc6c:/# /opt/influxdb/influx -host influx-mysql-cp.beget -database mysql-cp
Connected to http://influx-mysql-cp.beget:8086 version 0.9.6
InfluxDB shell 0.9.4.2
> show diagnostics
name: build
-----------
Branch  Build Time                      Commit                                          Version
0.9.6   2015-12-09T16:10:21+0000        ea8bbabfb68394aab5d9f23f08e29986bc137955        0.9.6


name: hh
--------
node    active  last modified   head    tail


name: network
-------------
hostname
logstorage.beget.ru


name: runtime
-------------
GOARCH  GOMAXPROCS      GOOS    version
amd64   12              linux   go1.4.2


name: system
------------
PID     currentTime                     started                         uptime
1061    2015-12-10T05:42:10.23260527Z   2015-12-09T23:04:12.638944394Z  6h37m57.593661219s

>
```


##### instance 2:
influxdb is built from commit 639f7f4bec98e39f47e2bac60f577b23b42910f8
input data is unmodified, with 2-field points in seria, e.g. 
```
mysql_quota,customer=fh7903p8,hostname=mars1,user=fh7903p8_wp-mysql blocks=352256i,files=35i 1449717000000000000)
```
uptime is 1.5 hour after upgrade
```
root@0edfceb9cc6c:/# /opt/influxdb/influx -host influx-quota.beget -database quota
Connected to http://influx-quota.beget:8086 version v0.9.6-rc1.n1449719628
InfluxDB shell 0.9.4.2
> show diagnostics
name: build
-----------
Branch  Build Time                      Commit                                          Version
HEAD    2015-12-10T03:54:25.558494      639f7f4bec98e39f47e2bac60f577b23b42910f8        v0.9.6-rc1.n1449719628


name: hh
--------
node    active  last modified   head    tail


name: network
-------------
hostname
logstorage.beget.ru


name: runtime
-------------
GOARCH  GOMAXPROCS      GOOS    version
amd64   12              linux   go1.4.3


name: system
------------
PID     currentTime                     started                         uptime
93      2015-12-10T08:43:47.397536601Z  2015-12-10T06:52:46.867854111Z  1h51m0.529682888s

>
```

the fix from 639f7f4bec98e39f47e2bac60f577b23b42910f8 seem to be working )
influxdb binary from master was crashing almost every write (data is being written every 10 minutes)

**Comment by mlosev** [comment]
```
name: system
------------
PID     currentTime                     started                         uptime
93      2015-12-10T18:32:02.470988881Z  2015-12-10T06:52:46.867854111Z  11h39m15.603135191s
```
no crash after upgrading to 639f7f4bec98e39f47e2bac60f577b23b42910f8
